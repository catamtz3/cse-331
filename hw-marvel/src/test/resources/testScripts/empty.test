LoadGraph empty empty.csv

FindPath empty n1 n2
