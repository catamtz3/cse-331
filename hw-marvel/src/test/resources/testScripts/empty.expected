loaded graph empty

path from n1 to n2:
no path found